app/src/main/java/io/gnosis/safe/ui/settings/view/NamedAddressItem.kt

```diff
@@ -21,7 +25,7 @@ class NamedAddressItem @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
-) : ConstraintLayout(context, attrs, defStyleAttr) {
+) : ConstraintLayout(context, attrs, defStyleAttr), Target {
```

Comment:
KnownAddressImageView could implement target and this way we can keep logo loading logic at one place

Reply:
Fixed in e578d2fdd6d4f255c3ae3378dfc5a6258ad87860